Tasks/CordovaBuild/lib/cordova-task.js

```diff
@@ -362,6 +362,9 @@ function execBuild(code) {
                 }
 
                 if (platform === 'android') {
+                    // Removes the old temp directory
+                    shelljs.rm("-rf", cordovaConfig.projectPath + "\\platforms\\android\\.gradle");
```

Comment:
@paymicro, please use `path.join()` or `path.resolve()` (preferable) for constructing filesystem paths